spec/features/auth_spec.rb

```diff
@@ -16,4 +16,46 @@
     expect(page).to have_content("user@example.com")
   end
 
+  scenario 'Users can go to register page' do
+    visit root_path
+    within("div.auth") { click_on "Register" }
+    expect(page).to have_content("Register")
+  end
+
+  scenario 'Users can go to register page' do
+    visit new_registration_path
+    within(".registration-form") { click_on "Register" }
+    expect(page).to have_content("Password can't be blank")
+  end
+
+  scenario 'Users can register' do
+    
+    visit new_registration_path
+    within(".registration-form") { click_on "Register" }
+    expect(page).to have_content("Password can't be blank")
+
+    fill_in "Name", with: "user"
+    fill_in "Email", with: "user@example.com"
+    fill_in "Password", with: "password"
+    fill_in "Confirm", with: "password"
+    within(".registration-form") { click_on "Register" }
+    expect(page).to have_no_content("Password can't be blank")
+
+  end
+
+  scenario 'User gets to new page opened after login registration' do
+
+    visit new_registration_path
+    within(".registration-form") { click_on "Register" }
+    expect(page).to have_content("Password can't be blank")
+
+    fill_in "Name", with: "user"
+    fill_in "Email", with: "user@example.com"
+    fill_in "Password", with: "password"
+    fill_in "Confirm", with: "password"
+    within(".registration-form") { click_on "Register" }
+    expect(page).to have_content("You are logged in successfully.")
+
+  end
```

Comment:
The last one is really all you needed, but the other ones are great for completeness and developing fluency with capybara and RSpec. Kudos.